page-factory-core/src/main/java/ru/sbtqa/tag/pagefactory/fragments/FragmentDataTableUtils.java

```diff
@@ -0,0 +1,69 @@
+package ru.sbtqa.tag.pagefactory.fragments;
+
+import gherkin.ast.DataTable;
+import gherkin.ast.Step;
+import gherkin.ast.TableCell;
+import gherkin.ast.TableRow;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.apache.commons.lang3.reflect.FieldUtils;
+
+public class FragmentDataTableUtils {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add a private constructor to hide the implicit public one. ([squid:S1118](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3AS1118))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=63&issues=AWSSY2K0sk71k7lwrek6&open=AWSSY2K0sk71k7lwrek6)

[ISSUE_KEY]: AWSSY2K0sk71k7lwrek6